mainParagraph.js

```diff
@@ -0,0 +1,33 @@
+// Main: child paragraph of root container
+// Provides a function to replace content in paragraph (see appendToParagraphXX)
+
+const ID_HTML_ROOT = "root-container";
+const GLOBAL_CONTAINER = document.querySelector(`#${ID_HTML_ROOT}`);
+
+const ID_HTML_SUBMIT_BUTTION = "submit-button";
+const ID_HTML_INPUT_DESCRIPTION = "input-description";
+const ID_HTML_INPUT_FIELD = "input-field";
+const ID_HTML_OUTPUT_DESCRIPTION = "output-description";
+const ID_HTML_PARAGRAPH_MAIN = "paragraph-main";
+const CLASS_NAME_HTML_FLEX_DIV = "flex-div";
+
+const HTML_G_PARAGRAPH_MAIN = document.createElement("paragraph");
+HTML_G_PARAGRAPH_MAIN.setAttribute("id", ID_HTML_PARAGRAPH_MAIN);
```

Comment:
there isn't a `paragraph` element in HTML, did you mean `p`? regardless, you are adding children to this element subsequently in your code, such as `div`s. for semantic HTML, a `p` element is probably not the best choice here, and a `div` would probably be better

Reply:
noted